ibis/expr/operations.py

```diff
@@ -2666,8 +2670,7 @@ class ExpressionList(Node):
     exprs = Arg(rlz.noop)
 
     def __init__(self, values):
-        values = list(map(rlz.any, values))
-        super(ExpressionList, self).__init__(values)
+        super(ExpressionList, self).__init__(list(map(rlz.any, values)))
```

Comment:
`ir.ExprList`, `ops.ExpressionList`, `ops.ValueList` are still ambiguous, should we disambiguate them in a subsequent PR?

Reply:
There's actually one more: `ir.ListExpr` :) The naming isn't great there, and here's my take on what each means.

- `ir.ExprList`: the expression type of `ops.ExpressionList`, representing a list of `Expr`s. In theory, this could represent things like an array of `Expr`s in backends that support it. It's unclear why a `list` wouldn't work for these cases except that `ExprList` provides the `execute` method. One important thing to note here is that `ExprList`s can have expressions of different types. See [this test]() for an example of that.
- `ir.ListExpr`: the expression type of `ops.ValueList`, representing a list of `ValueExpr`s *of the same type*. I gather that this is different from an array in that it is used to represent things like `(1, 2, 3)` (or at least at some point it was) as well as a convenient way to express a uniformly typed list of expressions.

Here's my proposal for cleaning this up:

1. I'm not convinced we need `ExprList`. If we do, then we need to document it better and differentiate it from `ValueList`
1. `ValueList` appears to be some combination of a set and an array. If there's a way we can potentially remove `ValueList` from the top-level API in favor of either one of those then we should do that.